global/prometheus-alertmanager-operated/values.yaml

```
@@ -35,6 +35,13 @@ slack:
   metal: {}
   network: {}
   vmware: {}
+  compute: {}
+  compute_storage_api: {}
+  email: {}
+  identity: {}
+  network_api: {}
+  observability: {}
+  containers: {}
```

you have specified a dict in ci/test-values.yaml already. the idea would be, to leave this entirely blank. This leads to a fail of the pipeline, in case the values are not in place, instead of deploying it with emtpy values. Linting then would fail too, in case the values are missing. You can incorporate the above lines too.

just go with this:
```suggestion
  compute:
  compute_storage_api:
  email:
  identity:
  network_api:
  observability:
  containers:
```